data/mainnet/tokens.js

```diff
@@ -5546,4 +5546,106 @@ module.exports = {
+    apyIconUrls: [],
+    apyTokenSymbols: ['BAL'],
+    displayName: 'Balancer: TUSD STABLE',
+    subLabel: 'TUSD',
+    tokenAddress: addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.bal_TUSD_STABLE.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.BALANCER,
+      params: [
+        addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+        addresses.MATIC.V2.bal_TUSD_STABLE.PoolId,
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.BALANCER,
+        params: [
+          addresses.MATIC.V2.bal_TUSD_STABLE.Underlying,
+          addresses.MATIC.V2.bal_TUSD_STABLE.PoolId,
+          '1000',
+          profitSharingCut8Percent,
+          CHAINS_ID.MATIC_MAINNET,
+        ],
+      },
+    ],
+    cmcRewardTokenSymbols: ['iFARM', 'BAL'],
+  },
+  bal_USDC_WETH_polygon: {
+    category: VAULT_CATEGORIES_IDS.BALANCER,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    isNew: true,
+    logoUrl: './icons/bal-usdc-eth.png',
+    apyIconUrls: [],
+    apyTokenSymbols: ['BAL'],
+    displayName: 'Balancer: USDC WETH polygon',
+    subLabel: 'USDC, WETH',
+    tokenAddress: addresses.MATIC.V2.bal_USDC_WETH_polygon.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.bal_USDC_WETH_polygon.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.BALANCER,
+      params: [
+        addresses.MATIC.V2.bal_USDC_WETH_polygon.Underlying,
+        addresses.MATIC.V2.bal_USDC_WETH_polygon.PoolId,
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.BALANCER,
+        params: [
+          addresses.MATIC.V2.bal_USDC_WETH_polygon.Underlying,
+          addresses.MATIC.V2.bal_USDC_WETH_polygon.PoolId,
+          '1000',
+          profitSharingCut8Percent,
+          CHAINS_ID.MATIC_MAINNET,
+        ],
+      },
+    ],
+    cmcRewardTokenSymbols: ['iFARM', 'BAL'],
+  },
+  bal_WBTC_WETH_polygon: {
+    category: VAULT_CATEGORIES_IDS.BALANCER,
+    chain: CHAINS_ID.MATIC_MAINNET,
+    isNew: true,
+    logoUrl: './icons/bal-wbtc-eth.png',
+    apyIconUrls: [],
+    apyTokenSymbols: ['BAL'],
+    displayName: 'Balancer: WBTC WETH polygon',
+    subLabel: 'WBTC, WETH',
+    tokenAddress: addresses.MATIC.V2.bal_WBTC_WETH_polygon.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.bal_WBTC_WETH_polygon.NewVault,
+    priceFunction: {
+      type: GET_PRICE_TYPES.BALANCER,
+      params: [
+        addresses.MATIC.V2.bal_WBTC_WETH_polygon.Underlying,
+        addresses.MATIC.V2.bal_WBTC_WETH_polygon.PoolId,
+        CHAINS_ID.MATIC_MAINNET,
+      ],
+    },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.BALANCER,
+        params: [
+          addresses.MATIC.V2.bal_WBTC_WETH_polygon.Underlying,
+          addresses.MATIC.V2.bal_WBTC_WETH_polygon.PoolId,
+          '1000',
```

Comment:
This number is wrong, please check out this public sheet where the reward amounts for the balancer pools are announced weekly: [https://docs.google.com/spreadsheets/d/1XDwShNDhKKzINtwBja7TsOw34JIwYktjGPLiG0WQl1s/edit?usp=sharing](https://docs.google.com/spreadsheets/d/1XDwShNDhKKzINtwBja7TsOw34JIwYktjGPLiG0WQl1s/edit?usp=sharing)